### ISSUE TITLE
DataStreams

### ISSUE THREAD

**Issue by tbreloff** [JuliaPlots/Plots.jl#53]
Implement interface of:  https://github.com/JuliaDB/DataStreams.jl

**Comment by mkborregaard** [comment]
Sounds interesting but I'm a little unsure what you're envisioning here?

**Comment by piever** [comment]
Closed by [#82](https://github.com/JuliaPlots/StatPlots.jl/pull/82). This now works

```julia
using RDatasets, CSV, StatPlots
iris = dataset("datasets","iris")
writetable("/tmp/iris.csv", iris)
@df CSV.Source("/tmp/iris.csv") scatter(:SepalLength, :SepalWidth)
```

and so should any DataStream.
